UnityProject/Assets/Scripts/ScriptableObjects/Research/Artifacts/Effects/AreaEffects/SicknessAreaEffect.cs

```diff
@@ -0,0 +1,33 @@
+using HealthV2;
+using Health.Sickness;
+using UnityEngine;
+
+namespace Systems.Research
+{
+	/// <summary>
+	/// Gives nearby players artifact sickness
+	/// </summary>
+	[CreateAssetMenu(fileName = "SicknessAreaEffect", menuName = "ScriptableObjects/Systems/Artifacts/SicknessAreaEffect")]
+	public class  SicknessAreaEffect : AreaEffectBase
+	{
+		[Tooltip("The sickness to infect nearby players with, uses index from SicknessManager")]
+		[SerializeField] private int sicknessindex;
```

Comment:
Thats not a good system, do we use SO's yet for sickness?

Reply:
Nah, they stored as gameobjects inside the sicknessmanager

Reply:
Can you change this to gameobject then for now instead of this index